packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -113,6 +114,39 @@ export class QuoteRequestor {
     private readonly _schemaValidator: SchemaValidator = new SchemaValidator();
     private readonly _orderSignatureToMakerUri: { [orderSignature: string]: string } = {};
 
+    public static makeQueryParameters(
+        takerAddress: string,
+        marketOperation: MarketOperation,
+        makerAssetData: string,
```

Comment:
Consider merging `makeQueryParameters()` with the existing `inferQueryParameters()` function (using whatever name feels appropriate to the unified concept).  Seems like it would be confusing to come into this code and see both of those functions and try to understand the difference between the two.

Reply:
great suggestion! I refactored